src/main/scala/org/broadinstitute/dsp/HelmInterpreter.scala

```diff
@@ -77,7 +78,8 @@ class HelmInterpreter[F[_]: ContextShift](blocker: Blocker, concurrencyBound: Se
 
   private def translateResult(cmd: String, result: String): F[Unit] = result match {
     case "ok" => logger.info(s"The command '${cmd}' succeeded")
-    case s    => F.raiseError(HelmException(s))
+    case "cannot re-use a name that is still in use" => logger.info(s"The command ${cmd} encounter a conflict. This is likely a resubmission of the same command. This will not raise an exception to ensure idempotency")
```

Comment:
Is this the literal string that helm returns? Wondering if we should do a substring match instead:
```
case s if s.contains("cannot re-use a name that is still in use") => logger.info("…")
```

Also as a future enhancement we could make the `main.go` methods return a status code/enum or something, instead of just a raw string.

https://github.com/broadinstitute/helm-scala-sdk/blob/master/helm-go-lib/main.go#L145

Reply:
Yeah,  it is.  I was going to do that if it was different for deleting and creating error msg, but since its the same and I'm  not too familiar with go I figured this was easier. I'll make a follow-up ticket